### PR TITLE
[linstor] Introduce `linstor-scheduler-admission.deckhouse.io/disabled` label

### PR DESCRIPTION
**Pull request by kvaps**
## Description

This PR adds a new label: `linstor-scheduler-admission.deckhouse.io/disabled`
If the label is set for a namespace, the pods inside this namespace will be excluded from `linstor-scheduler-admission` mutating webhook.

## Why do we need it, and what problem does it solve?

Get rid of `/tools/linstor_scheduler_webhook.go` generator in favor of the namespace label for excluding.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: linstor
type: chore
summary: Introduce linstor-scheduler-admission.deckhouse.io/disabled label
impact_level: default
```

### CODE REVIEW

File: dhctl/pkg/kubernetes/actions/manifests/manifests.go [hunk 1]
```diff
@@ -348,7 +348,8 @@ func DeckhouseNamespace(name string) *apiv1.Namespace {
 			Name: name,
 			Labels: map[string]string{
 				"heritage": "deckhouse",
-				"extended-monitoring.deckhouse.io/enabled": "",
+				"extended-monitoring.deckhouse.io/enabled":          "",
+				"linstor-scheduler-admission.deckhouse.io/disabled": "",
```
Comment (kvaps) [inline dhctl/pkg/kubernetes/actions/manifests/manifests.go:352]: Won't it be applied on existing cluster configuration?

**Review by yalosev — APPROVED** [review]

**Review by yalosev — APPROVED** [review]

File: dhctl/pkg/kubernetes/actions/manifests/manifests.go (hunk 1 above)
Comment (yalosev) [reply dhctl/pkg/kubernetes/actions/manifests/manifests.go:352]: It won't

**Review by yalosev — APPROVED** [review]

**Review by name212 — DISMISSED** [review]

File: modules/041-linstor/templates/linstor-scheduler-admission/webhook.yaml [hunk 2]
```diff
@@ -10,49 +8,13 @@ webhooks:
...
-      - d8-descheduler
-      - d8-ingress-nginx
-      - d8-log-shipper
-      - d8-pod-reloader
-      - d8-chrony
-      - d8-virtualization
-      - d8-cdi
-      - d8-okmeter
-      - d8-openvpn
+      - kube-system
```
Comment (apolovov) [inline modules/041-linstor/templates/linstor-scheduler-admission/webhook.yaml:17]: sure?

**Review by apolovov — APPROVED** [review]

**Review by shvgn — APPROVED** [review]

**Review by kvaps — COMMENTED** [review]
Needs to add linter and update internal documentation:
https://github.com/deckhouse/deckhouse/blob/main/docs/documentation/pages/internal/DEVELOPMENT.md

**Review by zuzzas — APPROVED** [review]

**Comment by apolovov** [comment]
We decided not to refactor this stuff.

**Pull request closed without merging**